docs/user-tutorial/data-diagnosis.md

```diff
@@ -164,8 +176,16 @@ The function used for this rule.
   For example, if the 'criteria' is `lambda x:x>0`, the rule is that if the raw data is larger than the 0, it should be defective.
 
 - `multi_rules`: the rule is to check if the combined results of multiple previous rules and metrics violate the criteria.
+  We would like to list several examples as follows:
+  - 'criteria: lambda label:True if label["rule4"]+label["rule5"]>=2 else False' means that this rule will be triggered if the sum of labeled metrics in rule4 and rule5 is larger than 2
+  - 'criteria: lambda label: bool(min(label["rule7"].values()))<1)' means that if the minimum of the metrics' values in rule6 is smaller than 1, it should be defective.
+    - If you reference a non-existent rule, it will raise exception
+    - If the test in the referenced rule failed or not run resulting in exception in creteria, it will not raise exception since it will be checked in failure_rule
```

Comment:
```suggestion
    - If you reference a non-existent rule, it will raise exception.
    - If the test in the referenced rule failed or not run resulting in exception in creteria, it will not raise exception since it will be checked in failure_rule.
```

Reply:
updated